react_juce/core/View.cpp

```diff
@@ -204,10 +206,155 @@ namespace reactjuce
 
         if (props.contains(backgroundColorProp))
         {
-            juce::Colour c = juce::Colour::fromString(props[backgroundColorProp].toString());
-
-            if (!c.isTransparent())
-                g.fillAll(c);
+            auto colorProp = props[backgroundColorProp];
+            //If object we assume it's a linear gradient
+            if(colorProp.isObject()) {
```

Comment:
Wowwww yes. Love this.

Thinking we might want to reuse this in many places... maybe we pull it out into a standalone function in our Utils.h/cpp file? The function could return a `std::variant<juce::Colour, juce::ColourGradient>` depending on what it learned from the object passed in.

Reply:
sweeet yup did have in the back of my mind that this logic will probably need to be reusable for a few other things we paint so good call. 
 
Have moved color variant logic over to utils, let me know if it's along line of what you were thinking.